evalml/utils/gen_utils.py

```diff
@@ -270,3 +271,16 @@ def is_all_numeric(df):
         if dtype not in numeric_and_boolean_dtypes:
             return False
     return True
+
+
+
+def _nullable_types_to_numpy_wrapper(pd_data):
+    """
+    Arguments:
+        df 
+    """
+    nullable_to_numpy_mapping = {pd.Int64Dtype: 'int64'}
```

Comment:
@angela97lin Here are the additional pandas dtypes.
https://github.com/pandas-dev/pandas/blob/0846dc1fdd8751492787f66b2e51cc1b168b5f20/pandas/__init__.py#L53
- Out of these, Woodwork uses CategoricalDtype, StringDtype, and BooleanDtype
- We will use Float64Dtype once the next release of pandas comes out.

Reply:
Thanks, @gsheni! This was super helpful :)